engine/gamesys/src/gamesys/test/test_gamesys.cpp

```diff
@@ -954,17 +954,7 @@ TEST_F(SpriteTest, FlipbookAnim)
 
     lua_State* L = scriptlibcontext.m_LuaState;
 
-    bool tests_done = false;
-    while (!tests_done)
-    {
-        ASSERT_TRUE(dmGameObject::Update(m_Collection, &m_UpdateContext));
-        ASSERT_TRUE(dmGameObject::PostUpdate(m_Collection));
-
-        // check if tests are done
-        lua_getglobal(L, "tests_done");
-        tests_done = lua_toboolean(L, -1);
-        lua_pop(L, 1);
-    }
+    WaitForTestsDone(10000, true, 0);
```

Comment:
To make it easier, I've created a helper function to wait for the result, and also correctly update+render the systems.

Previously, we only updated the systems, but never called into the render parts that contained the bug.